src/component.jl

```diff
@@ -448,17 +448,49 @@ end
 
 """
 Returns an iterator of Forecast instances attached to the component.
```

Comment:
Are you planning on importing these methods later in PSY?

Reply:
Yes.  For `System` we will have to change the existing function to be `IS.iterate_forecasts` and to forward the new parameters.